proknow-sdk/Patient/Entities/EntityScorecards.cs

```diff
@@ -73,7 +73,7 @@ public EntityScorecards(ProKnowApi proKnow, string workspaceId, string entityId)
             var customMetricIdsAndObjectives = resolvedCustomMetrics.Select(c => c.ConvertToScorecardSchema()).ToList();
 
             // Request the creation
-            var route = $"/workspaces/{_workspaceId}/entities/{_entityId}/metrics/sets";
+            var route = $"/workspaces/{_workspaceId}/patient/{_entityId}/metrics/sets";
```

Comment:
Are these `_entityId`s actually the patient ID? If not, is there a patient ID that you can use here instead?